stable/heapster/values.yaml

```diff
@@ -10,6 +10,7 @@ image:
 #   kubernetes.io/cluster-service: "true"
 #   kubernetes.io/name: "Heapster"
 labels:
+nodeSelector:
```

Comment:
Add empty braces (`{}`)  to indicate this is a map. And if you don't mind, add them to `labels:` too.